compositor_common/src/scene.rs

```diff
@@ -65,6 +65,7 @@ pub struct SceneSpec {
 pub struct InputSpec {
     pub input_id: InputId,
     pub resolution: Resolution,
+    pub fallback_color: Option<(u8, u8, u8)>,
```

Comment:
Maybe allow specifying alpha? Someone may want to use transparent
```suggestion
    pub fallback_color_rgba: Option<(u8, u8, u8, u8)>,
```

Reply:
we don't support alpha channel on inputs, because YUV does not support it

Reply:
If we `Clear` "input node" output texture (which is in RGBA format) with user-provided color, we can easily support alpha :)

Reply:
yes, but what would it mean? This is a fallback for input, if the input does not support the alpha channel, then supporting that in the alpha channel is a bit redundant.